packages/mobile/src/components/UploadedImage/UploadedImage.component.tsx

```diff
@@ -7,15 +7,18 @@ import FastImage from 'react-native-fast-image'
 export const UploadedImage: FC<UploadedImageProps> = ({ media, openImagePreview }) => {
   const { path, width, height } = media
 
+  const aspectRatio = width && height ? width / height : undefined
   const maxWidth = width && width >= 400 ? 400 : width
 
   return (
-    <View>
+    <View style={{ padding: 5 }}>
       {!path || !width || !height ? (
         <ActivityIndicator size='small' color={defaultTheme.palette.main.brand} />
       ) : (
         <TouchableWithoutFeedback onPress={() => openImagePreview(media)}>
-          <FastImage source={{ uri: `file://${path}` }} style={{ maxWidth, aspectRatio: width / height }} />
+          <View>
```

Comment:
This is the fix for displaying images.  Turns out `TouchableWithoutFeedback` doesn't like when child objects aren't wrapped in a view.